docs/docs/examples/guides/abb-hardware-interface.md

```diff
@@ -0,0 +1,292 @@
+the HTTPS protocol. The hardware interface uses RWS for auxiliary functionality, such as starting/stopping the program
+and the motors, and setting IOs. Setting up RWS in the simulator and the actual robot requires slightly different steps,
+which will be explained in the following sections. More information can be found in the
+[product reference page](https://developercenter.robotstudio.com/api/RWS).
+
+### RAPID
+
+RAPID is the programming language of ABB robots. Users can utilize RAPID to set up and execute their workflows and
+processes. This is enabled by user-defined libraries called **Modules**, that contain variables and functions or
+processes **(PROCs)**. Modules can then be loaded in controller **Tasks**, and called as required.
+
+<!-- :::note
+
+The example above drives the robot in velocity by setting the **PosCorrGain** parameter to 0 in the control loop. Remove
+this if the robot needs to receive position commands.
+
+::: -->
+
+## Mock interface
+
+The mock interface variant of different ABB manipulators can be used to check the application execution, visualize
+trajectory execution and ensure correct frame definition, among else. The mock interface is a very useful tool during
+early development stages, as it does not require connection to the actual hardware and provides a safe environment to
+experiment. After correct operation is verified, users can just switch to the real hardware interface to connect to the
+actual robot or simulator.
+
+:::note
+
+The mock interface does not include a physics engine, or dynamics calculations, and merely sets the commands as state.
+For more sophisticated simulation capabilities, check out RobotStudio in the following section.
+
+:::
+
+## Connecting to a robot
+
+The provided hardware interface can be used to connect and control either a simulated or a real robot. The following
+sections provide the necessary steps to do both.
+
+### RobotStudio simulation
+
+[RobotStudio](https://new.abb.com/products/robotics/software-and-digital/robotstudio) is the official ABB offline
+programming and simulation tool for robotics applications. It allows to run virtual controllers that mimic the behavior
+of the real robot, ensuring seamless transition between simulation and hardware. It can also be used to configure
+several aspects of the real controller.
+
+:::note
+
+The RobotStudio software suite is available only for Windows, so you may need to set it up on a secondary device or a
+virtual machine .
+
+:::
+
+Setting up a virtual workstation and controller is the next step after using the mock interface. This can be achieved by
+following the next steps:
+
+1. In RobotStudio, navigate to the **Add-Ins** tab and go to **Gallery**. There is a list of all available robot models
+   and RobotWare versions, the internal controller software. Make sure to install the versions present in the actual
```

Comment:
maybe replace "actual" with something saying "To ensure consistency between simulation and reality, install the versions matching the real robot controller, if one is available.